<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPost</class>
 <widget class="QWidget" name="EditPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(228, 228, 228);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>560</width>
     <height>265</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>请输入你的求助内容，140词以内</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_release">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(166, 200, 255);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>发  布</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(166, 200, 255);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
